test/WebJobs.Script.Tests/UtilityTests.cs

```diff
@@ -886,57 +886,51 @@ public void VerifyWorkerIndexingDecisionLogic(bool workerIndexingFeatureFlag, bo
         }
 
         [Theory]
-        [InlineData(true, true, false, "", true)]
-        [InlineData(true, true, true, "NonApp", true)]
-        [InlineData(true, true, true, "AppName", true)]
-        [InlineData(true, true, false, "NonApp", true)]
-        [InlineData(true, true, false, "AppName", true)]
-        public void VerifyWorkerIndexingFeatureFlagTakesPrecedence(bool workerIndexingFeatureFlag, bool workerIndexingConfigProperty, bool enabledHostingConfig, string disabledHostingConfig, bool expected)
+        [InlineData(true, true, "", true)]
+        [InlineData(true, true, "NonApp", true)]
+        [InlineData(true, true, "AppName", true)]
+        public void VerifyWorkerIndexingFeatureFlagTakesPrecedence(bool workerIndexingFeatureFlag, bool workerIndexingConfigProperty, string disabledHostingConfig, bool expected)
         {
-            VerifyCanWorkerIndexUtility(workerIndexingFeatureFlag, workerIndexingConfigProperty, enabledHostingConfig, disabledHostingConfig, expected);
+            VerifyCanWorkerIndexUtility(workerIndexingFeatureFlag, workerIndexingConfigProperty, disabledHostingConfig, expected);
         }
 
         [Theory]
-        [InlineData(true, false, false, "", false)]
-        [InlineData(true, false, true, "NonApp", false)]
-        [InlineData(true, false, true, "AppName", false)]
-        [InlineData(true, false, false, "NonApp", false)]
-        [InlineData(true, false, false, "AppName", false)]
-        [InlineData(true, true, false, "AppName", true)]
-        public void VerifyWorkerConfigTakesPrecedence(bool workerIndexingFeatureFlag, bool workerIndexingConfigProperty, bool enabledHostingConfig, string disabledHostingConfig, bool expected)
+        [InlineData(true, false, "", false)]
+        [InlineData(true, false, "NonApp", false)]
+        [InlineData(true, false, "AppName", false)]
+        [InlineData(true, true, "AppName", true)]
+        public void VerifyWorkerConfigTakesPrecedence(bool workerIndexingFeatureFlag, bool workerIndexingConfigProperty, string disabledHostingConfig, bool expected)
```

Comment:
This means that workerConfig is taking the topmost precedence in deciding `CanWorkerIndex()` even before featureFlag? Am I understanding this correctly?

Reply:
The title is a little misleading. For worker indexing to return true we need both feature Flag and worker config to return true. However, with this new change feature flag is no longer necessary. So worker config is the sole deciding factor if all other conditions are met.